### ISSUE TITLE
[sass|scss] Parse error with @Keyframes using capital K

### ISSUE THREAD

**Issue by nithinankam** [sasstools/sass-lint#902]
@keyframes #{pulsate}-#{$i} {
      0% {
        -webkit-transform: rotate($i) scale(1);
        -moz-transform: rotate($i) scale(1);
        -ms-transform: rotate($i) scale(1);
        -o-transform: rotate($i) scale(1);
        transform: rotate($i) scale(1);
      }
      50% {
        -webkit-transform: rotate($i) scale(1.5);
        -moz-transform: rotate($i) scale(1.5);
        -ms-transform: rotate($i) scale(1.5);
        -o-transform: rotate($i) scale(1.5);
        transform: rotate($i) scale(1.5);
      }
      100% {
        -webkit-transform: rotate($i) scale(1);
        -moz-transform: rotate($i) scale(1);
        -ms-transform: rotate($i) scale(1);
        -o-transform: rotate($i) scale(1);
        transform: rotate($i) scale(1);
      }
    }

What version of Sass Lint are you using?
V1.9.1
Actually, using https://www.npmjs.com/package/gulp-sass-lint which picks up 1.9.1
Please include any relevant parts of your configuration

What did you do? Please include the actual source code causing the issue.

What did you expect to happen?

What actually happened? Please include any error messages given to you by Sass Lint.

If you're using a IDE plugin have you tried the CLI too?

**Comment by DanPurdy** [comment]
Hi @nithin123456 This is an error with the AST we use [Gonzales-pe](https://github.com/tonyganch/gonzales-pe) could you please search for or report the issue there.

Thanks

**Comment by bgriffith** [comment]
Just a heads up that this fix https://github.com/tonyganch/gonzales-pe/pull/215 will solve this problem.

**Comment by tommycoppers** [comment]
@bgriffith @DanPurdy Would https://github.com/tonyganch/gonzales-pe/pull/215 fix issues like the following:

#### Errors

> **ERROR:** Please check validity of the block starting from line

Problem seems to be with the `--` after the interpolation.

```scss
#{$SELECTOR}--range {
  // ... styles
}
```

#### Valid 
```scss
// Workarounds
#{$SELECTOR}/--range {
  // ... styles
}

#{$SELECTOR}#{--range} {
  // ... styles
}

// Valid, but not the desired output
#{$SELECTOR}__range {
  // ... styles
}

#{$SELECTOR}-range {
  // ... styles
}
```

I'd be happy to make a ticket if this is a separate issue. 

Cheers!

**Comment by bgriffith** [comment]
Hey @tommycoppers - I can confirm that it's fixed in the latest branch of gonzales. So once a new version is released it'll get fixed. Hopefully not too long.

**Comment by tommycoppers** [comment]
Wonderful!  Thanks for the hard work!

> **VERIFIED!** I grabbed the dev branch and ran that locally with `sass-lint` and it worked like a charm.

**Comment by bgriffith** [comment]
@tommycoppers your issue is fixed in the latest release.

@nithinankam there is still an issue but this now appears to be down to the fact that your `@keyframe` rule starts with a capital `K` - this will be fixed in the next release.

**Comment by bgriffith** [comment]
These issues are now fixed `v1.11.1`
